src/VisualStudio/Core/Def/Implementation/EditAndContinue/ManagedHotReloadLanguageService.cs

```diff
@@ -20,7 +20,7 @@
 namespace Microsoft.VisualStudio.LanguageServices.EditAndContinue
 {
     [Shared]
-    [Export(typeof(IManagedEditAndContinueLanguageService))]
+    [Export(typeof(IManagedHotReloadLanguageService))]
```

Comment:
Just out of curiosity's sake, what does swapping these export definitions end up doing ?

Reply:
The previous interface was incorrect, so this didn't get imported by the debugger correctly.